MapboxNavigation/NavigationMapView.swift

```diff
@@ -457,8 +457,8 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             source.shape = polylines
             sourceSimplified.shape = mainPolylineSimplified
         } else {
-            let lineSource = MGLShapeSource(identifier: sourceIdentifier, shape: polylines, options: [.lineDistanceMetrics: true])
-            let lineCasingSource = MGLShapeSource(identifier: sourceCasingIdentifier, shape: mainPolylineSimplified, options: [.lineDistanceMetrics: true])
+            let lineSource = MGLShapeSource(identifier: sourceIdentifier, shape: polylines, options: nil)
```

Comment:
Maybe this could be a separate PR? It's not specific to offline routing and we probably want to be able to cite it (and easily revert it) separately